### PR TITLE
Create zh_cn, zh_tw, zh_hk translations

### PR DESCRIPTION
**Pull request by SolidBlock-cn**

### CODE REVIEW

File: src/main/resources/assets/effective/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "effective.subtitles.ambience.waterfall": "瀑布",
+  "text.autoconfig.effective.option.generateCascades": "联锁生成",
+  "text.autoconfig.effective.option.generateSplashes": "生成飞溅物",
```
Comment (ChuijkYahus) [inline src/main/resources/assets/effective/lang/zh_cn.json:4]:
```suggestion
  "text.autoconfig.effective.option.generateCascades": "生成瀑布粒子",
  "text.autoconfig.effective.option.generateSplashes": "生成飞溅粒子",
```

**Review by ChuijkYahus — COMMENTED** [review]
Also I think the lang json files are getting updated.
话说是不是该更新下了（）

**Comment by SolidBlock-cn** [comment]
OK

**Comment by qznfbnj** [comment]
联锁(Cascades)是个什么东西？连锁瀑布吗？
What's 联锁(Cascades)? Chain waterfall?

**Comment by doctor4t** [comment]
Sorry I missed this PR again, do you want to include all new lang entries from Effective 2.0 so I can merge it?

**Pull request merged**